projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -262,6 +262,11 @@ export class DatatableComponent implements OnInit, DoCheck, AfterViewInit {
    */
   @Input() loadingIndicator: boolean = false;
 
+  /**
+   * Show the loading spinner.
+   * Default value: `false`
+   */
+  @Input() spinnerIndicator: boolean = false;
```

Comment:
Am am not sure about this. I think we only need to loading indicator input and have an option to configure how it looks.

Reply:
loadingIndicator for default progress bar. if we want custom indicator, we can set customLoadingIndicator to true.